src/main/java/collections/Demo.java

```diff
@@ -0,0 +1,62 @@
+package collections;
+
+import java.util.*;
+
+public class Demo {
+    private static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        MovieArrayList moviesList = new MovieArrayList();
```

Comment:
Well, technically you've tested how collections work, but what's the reason to create your own wrappers for collections?
In general such kind of wrappers are rarely needed, mostly if you need some kind of transormation `mapping` logic

Reply:
@ssheludchenko Sorry, I mixed up the task. Fixed.